gapis/trace/android/profile/groups.go

```diff
@@ -117,3 +152,27 @@ func (n *groupTreeNode) findOrInsert(id int32, name string, link sync.SubCmdRang
 	slice.InsertBefore(&n.children, idx, groupTreeNode{id, name, link, nil})
 	return &n.children[idx], false
 }
+
+func (n *groupTreeNode) findOrInsertDrawCall(id int32, name string, link api.SubCmdIdx) (*groupTreeNode, bool) {
```

Comment:
is id here a command id? what is it identifying? Similarly what is name the name of? Is it the drawcall name?

Reply:
`groupTreeNode` has a field called `id` and `name`. These are the values for those fields in case it needs to be cretated.